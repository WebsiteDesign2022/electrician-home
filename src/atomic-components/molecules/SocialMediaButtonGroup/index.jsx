import React from "react";
import SocialMediaButton from "../../atoms/SocialMediaButton";

const SocialMediaButtonGroup = () => (
    <ul className="nav">
        <li className="nav-item">
            <SocialMediaButton name="Meta" link="#" />
        </li>
        <li className="nav-item">
            <SocialMediaButton name="Twitter" link="#" />
        </li>
        <li className="nav-item">
            <SocialMediaButton name="RSS" link="#" />
        </li>
    </ul>
);

export default SocialMediaButtonGroup;
