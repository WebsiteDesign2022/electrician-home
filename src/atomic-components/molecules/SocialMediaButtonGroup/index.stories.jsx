import React from 'react';

import SocialMediaButtonGroup from '.';

export default {
  title: 'Electrician-Home/molecules/SocialMediaButtonGroup',
  component: SocialMediaButtonGroup,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <SocialMediaButtonGroup {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {

};
