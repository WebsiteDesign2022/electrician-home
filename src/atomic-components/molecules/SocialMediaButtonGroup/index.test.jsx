import { render } from '@testing-library/react'

import SocialMediaButtonGroup from '.';

describe("SocialMediaButtonGroup", () => {
    test("renders SocialMediaButtonGroup and matches snapshot", () => {
        render(
            <SocialMediaButtonGroup />
        );

        expect(screen).toMatchSnapshot();

    });
});
