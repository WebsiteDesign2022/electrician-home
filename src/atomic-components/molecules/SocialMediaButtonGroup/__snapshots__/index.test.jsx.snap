// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialMediaButtonGroup renders SocialMediaButtonGroup and matches snapshot 1`] = `Screen {}`;
